NCrontab.Tests/CrontabScheduleTests.cs

```diff
@@ -343,6 +349,10 @@ static void BadField(string expression, bool includingSeconds)
             {
                 IncludingSeconds = includingSeconds
             }), Is.Null);
+            Assert.False(CrontabSchedule.TryParse(expression, new ParseOptions
+            {
+                IncludingSeconds = includingSeconds
+            }, out var _));
```

Comment:
For consistency with the rest of the tests, use Assert.That:

```suggestion
            Assert.That(CrontabSchedule.TryParse(expression, new ParseOptions
            {
                IncludingSeconds = includingSeconds
            }, out _), Is.False);
```